src/TrackGenerator.cpp

```diff
@@ -2036,3 +2049,38 @@ double TrackGenerator::getPhi(int azim) {
   else
     return 2 * M_PI - _phi[azim - _num_azim];
 }
+
+
+/**
+ * @brief Deletes the Timer's timing entries for each timed code section
+ *        code in the loop over track for ray tracing.
+ */
+void TrackGenerator::clearTimerSplits() {
+  _timer->clearSplits();
+}
+
+
+/**
+ * @brief Prints a report of the timing statistics to the console.
+ */
+void TrackGenerator::printTimerReport() {
+
+  std::string msg_string;
+
+  log_printf(TITLE, "TRACKGENERATOR TIMING REPORT");
+
+  /* Get the total runtime */
+  double tot_time = _timer->getSplit("Total time");
+  msg_string = "Total time for ray tracing";
+  msg_string.resize(53, '.');
```

Comment:
What do you think about having 53 be a constant, like `REPORT_WIDTH`? I'm putting some reports for memory and run parameters in ClosedMOC and am finding that I'm using this value a lot and a constant might be a little cleaner to use throughout.